packages/fhir-converter/src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -483,36 +517,14 @@ module.exports.external = [
     name: "evaluate",
     description: "Returns template result object: evaluate templatePath inObj",
     func: function (templatePath, inObj) {
-      try {
-        var getNamespace = require("cls-hooked").getNamespace;
-        var session = getNamespace(constants.CLS_NAMESPACE);
-        var handlebarsInstance = session.get(constants.CLS_KEY_HANDLEBAR_INSTANCE);
-        let templateLocation = session.get(constants.CLS_KEY_TEMPLATE_LOCATION);
-
-        var partial = handlebarsInstance.partials[templatePath];
-
-        if (typeof partial !== "function") {
-          var content = fs.readFileSync(templateLocation + "/" + templatePath);
-
-          // register partial with compilation output
-          handlebarsInstance.registerPartial(
-            templatePath,
-            handlebarsInstance.compile(content.toString())
-          );
-          partial = handlebarsInstance.partials[templatePath];
-        }
-        var result = partial(inObj.hash);
-        var processedResult = JSON.parse(jsonProcessor.Process(result));
-
-        // Check if the processedResult is undefined or an empty object
-        if (processedResult === undefined || (Object.keys(processedResult).length === 0 && processedResult.constructor === Object)) {
-          return undefined;
-        }
-
-        return processedResult;
-      } catch (err) {
-        throw `helper "evaluate" : ${err}`;
-      }
+      return evaluateTemplate(templatePath, inObj);
+    },
+  },
+  {
+    name: "optionalEvaluate",
```

Comment:
not currently being used, but the idea here is to be able to have a:

(#with evaluate xyz function) 

and if there is not return value from xyz function, to still execute the nested code